<launch>

    <arg name="footprint_pub" default="true"/>
    <arg name="odom_pub" default="true"/>

    <param name ="/base_frame" value ="base_link"/>
    <param name ='odometry_in' value ="/mavros/local_position/odom" />
    <param name="/send_odom" value="$(arg odom_pub)" />
    <param name="/send_footprint" value="$(arg footprint_pub)" />
    <param name="/odometry_out" value="/vehicle/odom" />
    <param name="/foorprint_out" value="/vehicle/foot_print" />
  
   <rosparam command="load" file="$(find vehicle_common)/params/vehicle_config.yaml" ns="vehicle"/>

  <param name="robot_description" command="$(find xacro)/xacro '$(find vehicle_common)/urdf/vehicle_description.xacro' "/>
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher">
  	<param name="publish_frequency" value="50"/>
  </node>

  <node pkg = "vehicle_common" type = "mavros_map_to_base.py" name = "tf_converter" />

</launch>
